plugins/modules/gcp_compute_instance.py

```diff
@@ -1112,6 +1118,7 @@ def main():
             state=dict(default='present', choices=['present', 'absent'], type='str'),
             can_ip_forward=dict(type='bool', aliases=['ip_forward']),
             deletion_protection=dict(type='bool'),
+            discard_local_ssd=dict(type='bool', required=False, default=True),
```

Comment:
default is true but the docs above don't have a default

Reply:
It's a newer feature, there's some docs [here](https://cloud.google.com/compute/docs/disks/local-ssd#stop_instance). The prior behavior was to delete always. Terraform Provider Google [always sets it to true](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance#nested_on_instance_stop_action).